youtube_dl/extractor/meipai.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from ..utils import parse_iso8601, unescapeHTML
+from .common import InfoExtractor
+
+# This is a sloppy fallback, except there is currently no other more reliable
+# ways to get the title if the page does not contain OpenGraph data, which
+# might be the case for some of the videos, the HTML output of the server is
+# not consistent.
+RE_TITLE = r'<title[^>]*>(.+)</title>'
+
+# This pattern will only showed up when this video is a playback of previous
+# live streaming
+RE_M3U8_URL = r'file:\s*encodeURIComponent\(["\'](.+)["\']\)'
+
+# This is the word in Simplified Chinese meaning 'Live Streaming Playback', it
+# will showed up in the keyword meta tag if the video is a recorded playback,
+# we use it to avoid false positives in the future as the RE_M3U8_URL pattern
+# is not specific enough.
+KEYWORD_PLAYBACK = '直播回放'
```

Comment:
There is no need in these constants since they are only used once. Some comments are also pointless since it's obvious from code.